packages/ketcher-react/src/script/ui/views/modal/components/document/Save/Save.jsx

```diff
@@ -94,6 +97,13 @@ class SaveDialog extends Component {
   componentDidMount() {
     const { checkOptions } = this.props.checkState
     this.props.onCheck(checkOptions)
+    if (this.formContainerRef.current) {
+      const {
+        width,
+        height
+      } = this.formContainerRef.current.getBoundingClientRect()
```

Comment:
Save dialog might have different dimensions depending on textarea size, and Save Image dialog picks up those dimensions and sets its width and height to avoid modal window resize on tab change.